### ISSUE TITLE
Add Gitcoin Newsletter Page

### ISSUE THREAD

**Issue by vs77bb** [gitcoinco/web#584]
## Task 
To grow the Gitcoin community more effectively, we'd like to house a Gitcoin Newsletter page on the Gitcoin website which we can link our Twitter + Slack audiences to, weekly, in case they are not subscribed to our weekly newsletter.  

## Description
- Build an HTML webpage at gitcoin.co/newsletter which shows the most recent version of the newsletter 
- Use [weekinethereum.com](http://www.weekinethereum.com/) as an example of how we'd like this to function (archive past newsletters at the bottom, include a link to sign up for the newsletter if you haven't already, etc.) 

Any questions? 

## Reviewers 
@owocki @vs77bb

**Comment by owocki** [comment]
does gitcoin.co/community come close enough to this already? PDF not HTML, and based upon github, not gitcoin.co... but still

**Comment by owocki** [comment]
another thought over the weekend about making the community content more accessible
```
owocki:
question for you all.. have you ever watched any of the livestreams after the fact?  they are posted at gitcoin.co/community but i dont think anyone really knows that.  i’d happily publish them to a video podcast if folks think that’d make them more consumable

Kind of an odd choice to ask a question in announcements... where we can't answer it... 

I recently shared the recorded stream with someone, and thought it could have used _a little_ editing maybe... but in general, yes, I think a podcast is a great idea.

With a little editing, most of them would make fine audio podcasts as well, which could broaden the consumer base.
Kevin Owocki
hey fokls — happy saturday!  i’m inside working for a bit while it’s cloudy out!  

question for you all.. have you ever watched any of the livestreams after the fact?  they are posted at gitcoin.co/community but i dont think anyone really knows that.  i’d happily publish them to a video podcast if folks think that’d make them more consumable
Thread in #community-announcmntsMar 10th at 1:53 PM


14 replies
Nick [24 hours ago]
I’ve watched a couple after the fact, but definitely if posted as a video podcast it would be easier


Mitch_Kosowski [23 hours ago]
it might make sense to have timed segments for topics written out in the google doc beforehand a la the tv show "Around the Horn" or "Pardon the Interruption"

10 minutes for this, 10 minutes for that, etc. no offense @e-n-l but I think you and I both tend to hog the microphone... would be good to keep us in check to make sure everyone gets a chance to participate! :thumbsup:


e-n-l [23 hours ago]
Yeah, that's definitely a real thing that I do.

e-n-l [23 hours ago]
I think I'm gonna need a chess timer.

Mitch_Kosowski [23 hours ago]
:joy: :joy: :joy:

Mitch_Kosowski [23 hours ago]
perhaps we could hook up tasers to a chess timer for us? that would make a pretty strong incenvitve :stuck_out_tongue:


e-n-l [23 hours ago]
I'm actually working on a project to do something like that.



e-n-l [23 hours ago]
Self-sovereign micromanagement


Mitch_Kosowski [23 hours ago]
interesting, using something like https://en.wikipedia.org/wiki/Pomodoro_Technique ?

i tried for a bit but found it too rigid... what works for me is to realize when i get stuck for a bit and then go take a walk as opposed to finding any set schedule (edited)


e-n-l [23 hours ago]
My MVP is a Christmas light timer that turns off my monitors, music and lamp. It's too rigid / dumb, so I have a raspberry Pi that will become a smart desk.


e-n-l [23 hours ago]
So yes, very similar


Mitch_Kosowski [22 hours ago]
cool, keep me posted on how it works for ya :thumbsup:


owocki [13 hours ago]
> Kind of an odd choice to ask a question in announcements... where we can’t answer it...


mah bad


owocki [13 hours ago]
> it might make sense to have timed segments for topics written out in the google doc beforehand a la the tv show “Around the Horn” or “Pardon the Interruption”


cc @vivek on this idea
```

**Comment by owocki** [comment]
@vs77bb i dont believe i'll hae the time to put in the legwork on your original task.. maybe we can ask someone to do this via a bounty?

**Comment by mkosowsk** [comment]
I can act as moderator if needed. Here are my qualifications:

0. Deadlift: 600 lbs
1. Squat: 500 lbs
2. Bench: 365 lbs (soon 405 lbs)

**Comment by owocki** [comment]
those numbers are no good here

**Comment by cmac2454** [comment]
I'm interested in this because I think it'd be great to see a summary of Gitcoin-related news on the site. I looked at the Week in Ethereum example, and I'm curious:

Are we looking to essentially do what they have, or do we want additional features/viewing options?

Would we consider pulling news from other sites? (e.g. Twitter, Week in Ethereum, etc.)

Do we want to make the newsletters downloadable as a PDF or other document if someone wants an offline copy of the news/sources?

I hope this gets developed, I think it'd be a solid addition to the site. I'd be interested in taking a crack at it myself too but I also don't want to inundate myself too much with work as I've already got a couple of tasks on my docket.

**Comment by owocki** [comment]
> Do we want to make the newsletters downloadable as a PDF or other document if someone wants an offline copy of the news/sources?


fyi -- this kind of already exists over at gitcoin.co/community

**Comment by owocki** [comment]
> I hope this gets developed, I think it'd be a solid addition to the site. I'd be interested in taking a crack at it myself too but I also don't want to inundate myself too much with work as I've already got a couple of tasks on my docket.

i wonder how we could narrow the scope here.. i mean the content already exists... its just a matter of finding a way to display it in a purdy way on the site

**Comment by vs77bb** [comment]
Closed for #640. cc @cmac2454 @owocki
